OpenNefia.Core/Input/Client/EngineContexts.cs

```diff
@@ -70,6 +73,7 @@ public static void SetupContexts(IInputContextContainer contexts)
             field.AddFunction(EngineKeyFunctions.ShowEscapeMenu);
             field.AddFunction(EngineKeyFunctions.QuickSaveGame);
             field.AddFunction(EngineKeyFunctions.QuickLoadGame);
+            field.AddFunction(EngineKeyFunctions.UIBacklog);
```

Comment:
Also, because this isn't a UI-related keybind, it should be titled `Backlog` or similar. It's most similar to the inventory keybinds so maybe it should be moved closer to them (instead of near the quick save/load keybinds, which are more "system-related").

Reply:
changed in 9eb449f